src/service/action-impl.js

```diff
@@ -86,8 +87,26 @@ export class ActionService {
     /** @const {!Window} */
     this.win = win;
 
+    /** @const @private {!Object<string, function(!ActionInvocation)>} */
+    this.actionHandlers_ = {};
+
+    /** @param {!Vsync} */
+    this.vsync_ = vsyncFor(this.win);
+
     // Add core events.
     this.addEvent('tap');
+
+    // Add core actions.
+    this.addAction('hide', invocation => {
+      // TODO(dvoytenko): Consider other ways to do "hide", including classes
+      // and animations. Or we can leave this up to the future "toggleClass".
+      // TODO(dvoytenko): Possibly Resources has to be notified to stop some
+      // AMP elements.
+      this.vsync_.mutate(() => {
+        invocation.target.style.display = 'none';
```

Comment:
Does it make more sense to go all in with toggleClass, addClass and removeClass?